coalib/coala_main.py

```diff
@@ -57,7 +56,8 @@ def run_coala(console_printer=None,
                                     `force_show_patch` will be ignored.
     :param force_show_patch:        If set to True, a patch will be always
                                     shown. (Using ApplyPatchAction.)
-    :param arg_parser:              Instance of ArgParser that is used to parse
+    :param arg_parser:              Instance of ``argparse.ArgumentParser``
+                                    that is used to parse
                                     non-setting arguments.
```

Comment:
Try to better distribute text over several lines so that lines are more filled up to allowed line length